lib/hexo/multi_config_path.js

```diff
@@ -22,12 +22,24 @@ module.exports = ctx => function multiConfigPath(base, configPaths) {
 
   } else {
     // only one config
+    let configPath = defaultPath;
+
+    // check for absolute config file
+    if (pathFn.isAbsolute(configPaths)) {
```

Comment:
It can be simplified to
```
configPath = pathFn.isAbsolute(configPaths) ? configPaths ? pathFn.resolve(base, configPaths);
```
then we can check `fs.existsSync` in a single branch.

Reply:
Nice add. Looks good to me.

Reply:
@JLHwung do I need to implement these changes on my branch? Or can you do it as a collaborator directly via GitHub?